### PR TITLE
WT-7783 Fix restore tombstone assert and condition for non timestamp tables

### PR DESCRIPTION
**Pull request by kommiharibabu**

### CODE REVIEW

File: src/txn/txn_rollback_to_stable.c [hunk 1]
```diff
@@ -508,15 +508,18 @@ __rollback_ondisk_fixup_key(WT_SESSION_IMPL *session, WT_REF *ref, WT_PAGE *page
...
+            if (!__rollback_check_if_txnid_non_committed(session, hs_tw->stop_txn) &&
+              hs_stop_durable_ts <= rollback_timestamp) {
                 /*
-                 * The restoring tombstone timestamp must be less than previous update start
+                 * The restoring tombstone timestamp must be zero or less than previous update start
                  * timestamp or the on-disk update is an out of order prepared.
                  */
-                WT_ASSERT(session, hs_stop_durable_ts < newer_hs_durable_ts || unpack->tw.prepare);
+                WT_ASSERT(session,
+                  hs_stop_durable_ts == 0 || hs_stop_durable_ts < newer_hs_durable_ts ||
```
Comment (quchenhao) [inline src/txn/txn_rollback_to_stable.c:521]: WT_TXN_NONE?

Comment (kommiharibabu) [reply src/txn/txn_rollback_to_stable.c]: Fixed by using WT_TS_NONE as it is a timestamp.

**Review by quchenhao — APPROVED** [review]
LGTM

**Review by hbokhari — APPROVED** [review]
LGTM.

**Pull request merged**